building_blocks/recipes.lua

```diff
@@ -20,15 +20,15 @@ end
 minetest.register_craft({
 	output = 'building_blocks:Adobe 3',
 	recipe = {
-		{"default:sand"},
-		{"default:clay"},
+		{homedecor.materials.sand},
+		{homedecor.materials.clay_lump},
```

Comment:
you could make things a little more legible if you put
```lua
    local materials = homedecor.materials
```
at the start of the file. not a big deal tho.

Reply:
technically locals are slightly faster, but as this is load time, doesnt really matter. leaving this open as undecided currently

Reply:
fixed https://github.com/mt-mods/homedecor_modpack/pull/28/commits/d8b96266a8f0c9c581b3e38bb01522ad66439ea7